src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -39,12 +40,14 @@ export class RundownTimelineService implements RundownService {
   public async activateRundown(rundownId: string): Promise<void> {
     await this.assertNoRundownIsActive()
     const rundown: Rundown = await this.rundownRepository.getRundown(rundownId)
-
+    const snapBefore = rundown.getInfinitePiecesMap()
```

Comment:
I would name these constants something along the line of `infinitePiecesBeforeActivation`. Could you add types to these constants as well? :-)